tests/notification/conftest.py

```diff
@@ -0,0 +1,38 @@
+"""Conftest file for notification tests."""
+
+from typing import Any
```

Comment:
nit: `Any` is the duct tape for python typing. We really should try to identify the types for cleanliness, security, and to hint to the IDEs for code completion.

FWIW:
```python
from selenium.webdriver.firefox.webdriver import WebDriver # for `selenium`
from pytest import Config, Parser # for `pytestconfig` & `firefox_options`
```

Reply:
I concur, I didn't put much effort here as these tests are pretty low risk, but I can add all the types if you would like that.